tests/unit/test_databricks.py

```diff
@@ -233,6 +233,44 @@ def test_replace_different_column_schema(databricks):
     databricks.drop_schema(TEST_SCHEMA)
 
 
+def test_snakecase_column_names(databricks):
+
+    assert not databricks._check_if_table_exists(schema=TEST_SCHEMA, table=TEST_TABLE)
+
+    origin_to_df = databricks.to_df.__wrapped__
```

Comment:
1) origin is a noun that means something else than original;
2) it's not possible to know what "original" would mean here or why you're doing this instead of just using `to_df()`; this begs a comment and possibly a more explicit variable name, such as `to_df_no_metadata_cols()`.

Reply:
[✅ Updated function name](https://github.com/dyvenia/viadot/pull/673/commits/4d28f6c1f51e2b9757186f6d3310cdca43a5d552)
[✅ Added comment to variable](https://github.com/dyvenia/viadot/pull/673/commits/250576c92b9a4072a7e6378d00a7419b55bbf9ec)